code/modules/admin/panels/player_panel.dm

```diff
@@ -466,14 +466,15 @@
 		<a href='?src=[ref];transform=combat_robot;mob=[REF(M)]'>Combat_Robot</a> |
 		<a href='?src=[ref];transform=monkey;mob=[REF(M)]'>Monkey</a> |
 		<a href='?src=[ref];transform=moth;mob=[REF(M)]'>Moth</a> |
-		<a href='?src=[ref];transform=zombie;mob=[REF(M)]'>Zombie</a> |
```

Comment:
Whats up with these l changes

Reply:
In the player panel, the buttons are supposed to have a `|` if they're not the last button. Most were missing it, and I just wanted to add that. If it's preferred though, I could just not do that.